packages/core/src/renderTexture/GenerateTextureSystem.ts

```diff
@@ -2,27 +2,26 @@ import { extensions, ExtensionType } from '@pixi/extensions';
 import { Matrix, Transform } from '@pixi/math';
 import { RenderTexture } from './RenderTexture';
 
-import type { MSAA_QUALITY, SCALE_MODES } from '@pixi/constants';
+import type { MSAA_QUALITY } from '@pixi/constants';
 import type { ExtensionMetadata } from '@pixi/extensions';
 import type { Rectangle } from '@pixi/math';
 import type { IRenderableContainer, IRenderableObject, IRenderer } from '../IRenderer';
 import type { ISystem } from '../system/ISystem';
+import type { IBaseTextureOptions } from '../textures/BaseTexture';
 
 const tempTransform = new Transform();
 
 // TODO could this just be part of extract?
-export interface IGenerateTextureOptions
+export interface IGenerateTextureOptions extends IBaseTextureOptions
 {
-    /** The scale mode of the texture. Optional, defaults to `PIXI.BaseTexture.defaultOptions.scaleMode`. */
-    scaleMode?: SCALE_MODES;
```

Comment:
Why did you remove `scaleMode` from these options?

Reply:
I made `IGenerateTextureOptions` extend `IBaseTextureOptions`.

Reply:
Oops missed that!. Thanks for clarifying